### DISCUSSION TITLE
Feature request: Allow the player to set a priority for different types of transport at a station

### DISCUSSION THREAD

**Discussion by Mantades** [OpenTTD/OpenTTD#11394] [Features]
The way I see it is like this:
If a station combines different means of transport (like trains + road vehicles + ships), then a player should be able to set a different priority for each type of transport. 
For example: 
Right now when a train is waiting at the station to get full and a batch of passengers shows up, then the train reserves all of them, even though the loading process takes some time. In the mean time a ship or a bus comes to the station (without the order to fully load) and leaves with nothing. 
With priorities set differently, the new coming bus or ship would load anything that was still not loaded on the train and continue it's journey further.

Of course it would make most sense while playing vanilla game, with manual cargo distribution.

**Comment by PeterN** [comment]
You could try turning off the `improved_load` setting.

**Reply by Mantades** [reply]
That would solve one case, which was just an example. There are other uses for such a feature. 
Another example would be a station where I want to prioritize other means of transportation and have a single (or more then a single) train just to collect whats left. It should stop loading whenever someone else shows up.
